lesson_17/tests/test_human.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from lesson_17.Human.human import Human
+
+
+@pytest.mark.parametrize("name,age,gender", [('Ann', 1, "female"), ('Bob', 99, "male")])
+@pytest.mark.positive
+def test_create_human(name, age, gender):
+    human = Human(name, age, gender)
+    assert hasattr(human, '_Human__status')
+    assert hasattr(human, '_Human__age_limit')
+
+
+@pytest.mark.positive
+def test_change_gender(create_human_positive):
+    # Change to a different gender
+    new_gender = "male" if create_human_positive.gender == "female" else "female"
+    create_human_positive.change_gender(new_gender)
+    assert create_human_positive.gender == new_gender
+
+
+@pytest.mark.negative
+def test_change_to_existed_gender(create_human_positive):
+    # Try to change to the same gender
+    new_gender = "male" if create_human_positive.gender == "male" else "female"
+    with pytest.raises(Exception):
+        create_human_positive.change_gender(new_gender)
+
+
+@pytest.mark.positive
+def test_grow_human_below_limit(create_human_positive):
+    for number in range(create_human_positive.age, 99):
```

Comment:
I do not think that verifying all available ages is a good idea. 
Just use Boundary Value Analysis